python/cugraph/cugraph/gnn/data_loading/bulk_sampler_io.py

```diff
@@ -69,7 +69,7 @@ def _write_samples_to_parquet(
         results_p["batch_id"] = offsets_p.batch_id.repeat(
             cupy.diff(offsets_p.offsets.values, append=end_ix)
         ).values
-        results_p.to_parquet(full_output_path)
+        results_p.reset_index(drop=True).to_parquet(full_output_path, compression=None)
```

Comment:
```suggestion
        results_p.to_parquet(full_output_path, compression=None, index=False)
```

Reply:
Ahh, cool. Did not know that we can drop index like this. Agreed on this being the better approach.